<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookQueryWindow</class>
 <widget class="QWidget" name="BookQueryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书查询</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayoutFilters">
     <property name="horizontalSpacing">
      <number>16</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelBookName">
       <property name="text">
        <string>书名包含：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditBookName">
       <property name="minimumWidth">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAuthor">
       <property name="text">
        <string>作者包含：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAuthor"/>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="layoutButtons">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReset">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="filterSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetBooks">
     <column>
      <property name="text">
       <string>书号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版社</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>库存</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInfo">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>提示：查询结果将实时刷新。</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBorrow">
       <property name="text">
        <string>借阅所选图书</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
